easybuild/easyblocks/t/tensorflow.py

```diff
@@ -222,6 +222,12 @@ def extra_options():
             'with_mkl_dnn': [with_mkl_dnn_default, "Make TensorFlow use Intel MKL-DNN", CUSTOM],
             'with_xla': [None, "Enable XLA JIT compiler for possible runtime optimization of models", CUSTOM],
             'test_script': [None, "Script to test TensorFlow installation with", CUSTOM],
+            'test_targets': [[], "List of Bazel targets which should be run during the test step", CUSTOM],
+            'test_tag_filters': ['', "Comma-separated list of tags to filter for during the test step", CUSTOM],
+            'test_jobs': [None, "Number of test jobs to run in parallel. Use None to automatically determine a value",
```

Comment:
Should perhaps add that None is the default in the help text here.

Reply:
Done